.github/workflows/gosec.yml

```diff
@@ -16,22 +20,15 @@ jobs:
     steps:
       - name: Checkout Source
         uses: actions/checkout@v3
-      - name: Get Diff
```

Comment:
this would run on pull_requests as well, could we keep that behaviour?

Reply:
@marbar3778 Pardon, didn't get it. You want to add these paths in pull requests as well?